### ISSUE TITLE
508-defect-3: Start without signing in link should be written as a link `a` , not a `button`

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#42237]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## User Story or Problem Statement

As a user, I expect to be able to open a link in a new tab so that I can navigate in a way that works best for me.

## Details

The "Start without signing in" link is written as a `button` which is materially dishonest. Users may become frustrated that they are unable to open a new tab. Screen reader users may be confused that the text seems more like a link despite being announced as a button.

For more on this long-tail effort, check out our [buttons vs. links Mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1610052905994/253723ef4b2ee9a726f538876a257df94d468db1) and this [Slack thread discussing _why_ material honesty is so important](https://dsva.slack.com/archives/C8E985R32/p1610659601009200?thread_ts=1610652897.002600&cid=C8E985R32).

"A few years at an accessibility event, a blind man was telling a story about how he was on the phone with someone who was trying to help him sign up for a service. The person on the phone told him to click the “sign up button” and he couldn’t find it because it was a link. He told the story with a lot more gravity than I can, but that was the point where I realized how important the human-human communication part of accessibility is, it’s not just machine to human." - Tim Wright

## Acceptance Criteria

- [ ] "Start your application without signing in" is written with an `a` tag 
- [ ] Design reviews and validates after accessibility suggestions are implemented

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/burials-and-memorials/application/530/introduction` in browser
2. Right click on "Start your application without signing in"
3. Confirm you cannot open in a new tab (it's a button)

## Solution (if known)

Use `a` instead of `button` 

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)

## Screenshots or Trace Logs

<img width="504" alt="Screen Shot 2022-06-02 at 3 03 08 PM" src="https://user-images.githubusercontent.com/14154792/171715599-36c2fd96-d937-44ae-8f36-dd8af720d55f.png">

**Comment by jerekshoe** [comment]
We might want to use the primary action link

**Comment by jaredcunha** [comment]
Here's the fix I have:

**Before:**
```jsx
<button
  className="va-button-link schemaform-start-button"
  onClick={this.goToBeginning}
  aria-label={ariaLabel}
  aria-describedby={ariaDescribedby}
  >
      Start your {appType} without signing in
</button>
```

**After:**
```jsx
import { withRouter, Link } from 'react-router';
...
<Link
  onClick={this.goToBeginning}
  to={this.getStartPage}
  className="vads-c-action-link--blue"
  aria-label={ariaLabel}
  aria-describedby={ariaDescribedby}
  >
     Start your {appType} without signing in
</Link>
```
<img width="681" alt="Screen Shot 2022-06-22 at 2 50 45 PM" src="https://user-images.githubusercontent.com/1094951/175114190-d68c37a2-15fb-43fc-8908-54c1a0db0e30.png">

This is is not actually limited to the `burials` application. It is in the `SaveInProgressIntro` component, which is imported into:
- `appeals`
- `ask-a-question`
- `coronavirus-research`
- `disability-benefits`
- `edu-benefits`
- `financial-status-report`
- `fry-dea`
- `hca`
- `lgy`
- `pensions`
- `my-education-benefits`
- `personalization`
- `pre-need`
- `sah`
- `vre`
- `health-care-questionnaire`
- Other mock forms, daily product scan things, etc.

It's used in a total 44 `.jsx` files, and that number doesn't any of the unit test files.


Additionally in benefits, when signed in, there is also a button that probably should be a link. I don't know if that should be part of this work. Seems as straightfoward as the fix above. I'll post a screenshot when the sign in is working again (currently down)

@jerekshoe @joshkimux

**Comment by joshkimux** [comment]
This is awesome @jaredcunha ! 🥳 

Couple of thoughts:
- We may want to consider using the link as it stands right now (instead of an action link) given that it's the less prioritized action (I know we want to typically nudge folks to log in first)
- As for the signed in experience, the button should indeed be a link-- and in this case probably an action link (if I have the right image in my head 🤔 )

Thank you both again for the super fast response! Y'all rock 💯

**Comment by jaredcunha** [comment]
Ah yeah @joshkimux  - I think I like it better without the action link, just the plain text when it's a secondary action. For the primary action… it's a green button, and I saw those were going to be deprecated I think outside of a super narrow use case. Do you have an idea if we should use the green or blue `ActionLink` for those?

**Comment by joshkimux** [comment]
We should use the [green `ActionLink`](https://design.va.gov/components/link/action#primary) given its the primary call to action on the page 😄

**Comment by jaredcunha** [comment]
> We should use the [green ActionLink](https://design.va.gov/components/link/action#primary) given its the primary call to action on the page 😄

@joshkimux and I spoke about this today and decided it's best, given some of the complexity in the click handling, we should tackle this in another ticket.
